src/main/kotlin/com/cognifide/gradle/aem/pkg/PrepareTask.kt

```diff
@@ -28,8 +31,17 @@ open class PrepareTask : AemDefaultTask() {
     fun prepare() {
         copyContentVaultFiles()
         copyMissingVaultFiles()
+        createEmptyJcrRoot();
+    }
+
+    private fun createEmptyJcrRoot() {
+        if (jcrRoot.exists()) {
+            jcrRoot.deleteRecursively()
+        }
+        jcrRoot.mkdir()
```

Comment:
`mkdir` vs `mkdirs` / what will be more sure?

Reply:
It's single folder under build/aemPrepare

Reply:
I was more thinking if plular version could make it more defensive. nvm